cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -685,9 +685,21 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			// this field has "genconversion=false" comment to ignore it.
 			continue
 		}
+		t := m.Type
+		if t.Kind == types.Alias {
+			copied := *t.Underlying
```

Comment:
Do we want to support multiple layers of aliasing? I mean if the "underlying" is again alias to something?